app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java

```diff
@@ -290,13 +289,13 @@ public void onActionModeFinished(ActionMode mode) {
   private void readAloudSelection(Menu menu) {
     if (menu != null) {
       menu.findItem(R.id.menu_speak_text)
-          .setOnMenuItemClickListener(item -> {
-            tts.readSelection(getCurrentWebView());
-            if (actionMode != null) {
-              actionMode.finish();
-            }
-            return true;
-          });
+              .setOnMenuItemClickListener(item -> {
+                tts.readSelection(getCurrentWebView());
+                if (actionMode != null) {
+                  actionMode.finish();
+                }
+                return true;
+              });
```

Comment:
Please don't make unrelated changes.

Reply:
Use 2 space for tabs, 4 space for line continuations.

Reply:
Whenever i am making a pull this is how this function looks in android studio

````
private void readAloudSelection(Menu menu) {
        if (menu != null) {
            menu.findItem(R.id.menu_speak_text)
                    .setOnMenuItemClickListener(item -> {
                        tts.readSelection(getCurrentWebView());
                        if (actionMode != null) {
                            actionMode.finish();
                        }
                        return true;
                    });
        }
    }`
```

Reply:
And i think this is how it should be - 
```
private void readAloudSelection(Menu menu) {
        if (menu != null) {
            menu.findItem(R.id.menu_speak_text)
                .setOnMenuItemClickListener(item -> {
                      tts.readSelection(getCurrentWebView());
                      if (actionMode != null) {
                          actionMode.finish();
                      }
                      return true;
                  });
        }
 }

```
But i have to manually do this everywhere....is there any way to do it automatically